app/components/Game/Game.js

```diff
@@ -16,34 +19,71 @@ export default function Game({
     endButtonPressed,
     goButtonPressed,
   },
-  handleClicks,
+  handleClicks: {
+    handleClickOnBoard,
+    toggleStartButtonPressed,
+    toggleEndButtonPressed,
+    findAndDrawPath,
+  },
 }) {
-  const gameHeaderState = {
+  const instructionsState = {
     pathStartingPoint,
     pathEndingPoint,
     startButtonPressed,
     endButtonPressed,
   };
-  const gameScreenState = {
+  const instructionsProps = { instructionsState, };
+  const instructions = (
```

Comment:
If the value is a react component, ItShouldBeCapitalized

Reply:
Ok, so here I'm a little confused. 

if i'd put the wrapper `<div>` inside the `<GameInstructions>` then this line would be:

`const Instructions = <GameInstructions { ...instructionsProps } >`

with the possibility of changing it to `const GameInstructions = ...` although I doubt I could do it because I already import `GameInstructions`. 

but, the value of putting the jsx structure inside a single word variable is that it will be much more comfortable for use later on.

Although, I could also go for:
`const gameHeaderChildren = [ <GameInstructions { ...props } />, <Link { ...otherProps } /> ]`

your comment from below about the fine line is very strongly felt..

in the meantime changed to `Instructions`